File: JAVA/Program/KategoriProduk.java
public class KategoriProduk extends Produk {
    protected String idKategori, namaKategori;

    public KategoriProduk(String idProduk, String namaProduk, int harga, int stok,
                          String idKategori, String namaKategori) {
        super(idProduk, namaProduk, harga, stok);
        this.idKategori = idKategori;
        this.namaKategori = namaKategori;
    }

    @Override
    public void tampilkanInfo() {
        super.tampilkanInfo();
        System.out.println("Kategori  : " + namaKategori + " (" + idKategori + ")");
    }
}

File: PYTHON/Program/KategoriProduk.py
from Produk import Produk

class KategoriProduk(Produk):
    def __init__(self, id_produk, nama_produk, harga, stok, id_kategori, nama_kategori):
        super().__init__(id_produk, nama_produk, harga, stok)
        self.id_kategori = id_kategori
        self.nama_kategori = nama_kategori

    def tampilkan_info(self):
        super().tampilkan_info()
        print(f"Kategori  : {self.nama_kategori} ({self.id_kategori})")
